pkg/resources/server.go

```diff
@@ -357,7 +360,10 @@ func (m *resourceServer) NotifyRegistrationStatus(ctx context.Context, regstat *
 	return &registerapi.RegistrationStatusResponse{}, nil
 }
 
-func getRdmaDeviceSpec(pciAddress string) []*pluginapi.DeviceSpec {
+type rdmaFinder struct {
```

Comment:
consider moving RdmaFinder and its implementation into its own package for cleaner code partitioning.